### PR TITLE
Update code format per new style

### PR DESCRIPTION
**Pull request by jstumbaugh**
Changes `lib/iex/test/iex/helpers_test.exs` for the new code format.

### CODE REVIEW

File: lib/iex/test/iex/helpers_test.exs [hunk 1]
```diff
@@ -742,24 +785,28 @@ defmodule IEx.HelpersTest do
...
+        elixirc(["sample.ex"])
 
         assert l(Sample) == {:module, Sample}
-        assert_raise UndefinedFunctionError, "function Sample.run/0 is undefined or private", fn ->
-          Sample.run
-        end
-      end
+
+        assert_raise UndefinedFunctionError,
+                     "function Sample.run/0 is undefined or private",
```
Comment (josevalim) [inline lib/iex/test/iex/helpers_test.exs]: Please extract the message to a variable.

File: lib/iex/test/iex/helpers_test.exs [hunk 2]
```diff
@@ -771,56 +818,64 @@ defmodule IEx.HelpersTest do
...
+        r(:nonexistent_module)
       end
     end
 
     test "reloads elixir modules" do
-      assert_raise UndefinedFunctionError, ~r"function Sample.run/0 is undefined \(module Sample is not available\)", fn ->
-        Sample.run
-      end
+      assert_raise UndefinedFunctionError,
+                   ~r"function Sample.run/0 is undefined \(module Sample is not available\)",
```
Comment (josevalim) [inline lib/iex/test/iex/helpers_test.exs]: Please extract the message to a variable.

File: lib/iex/test/iex/helpers_test.exs [hunk 3]
```diff
@@ -771,56 +818,64 @@ defmodule IEx.HelpersTest do
...
+      with_file(filename, test_module_code(), fn ->
+        assert capture_io(:stderr, fn ->
+                 assert c(filename, ".") == [Sample]
+                 assert Sample.run() == :run
+
+                 File.write!(filename, "defmodule Sample do end")
+                 assert {:reloaded, Sample, [Sample]} = r(Sample)
+
+                 assert_raise UndefinedFunctionError,
+                              "function Sample.run/0 is undefined or private",
```
Comment (josevalim) [inline lib/iex/test/iex/helpers_test.exs]: Please extract the message to a variable.

File: lib/iex/test/iex/helpers_test.exs (hunk 1 above)
Comment (jstumbaugh) [reply lib/iex/test/iex/helpers_test.exs]: Good catch, it has been fixed.

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**